src/main/scala/org/broadinstitute/dsde/firecloud/webservice/LibraryApiService.scala

```diff
@@ -47,6 +47,13 @@ trait LibraryApiService extends HttpService with FireCloudRequestBuilding
     } ~
     pathPrefix("api") {
       requireUserInfo() { userInfo =>
+        path("duos" / "consent" / "orsp" / Segment) { (orspId) =>
+          get { requestContext =>
+            perRequest(requestContext,
+              LibraryService.props(libraryServiceConstructor, userInfo),
+              LibraryService.SearchOrspId(orspId))
```

Comment:
@davidangb - I wonder if by making this an actor call that goes through a DAO, I'm making this more complex than it needs to be. I am thinking of making this a passthrough which would likely simplify the code a lot and also allow me to get rid of the exceptions in the tests by using MockServer.